src/main/java/frc/robot/commands/RotateTurret.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Limelight;
+import frc.robot.subsystems.Turret;
+
+public class RotateTurret extends CommandBase {
+
+    private Turret turret;
+    private Limelight lime;
+
+    public RotateTurret() {
+        addRequirements(turret, lime);
+    }
+
+    @Override
+    public void execute() {
+        lime.turnOn();
+        if (lime.canSeeTarget() && lime.getHorizontalAngle() > 1.5) {
+            turret.set(lime.getHorizontalAngle() > 0 ? 0.5 : -0.5);
+        }
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        if (Math.abs(lime.getHorizontalAngle()) <= 1.5) {
```

Comment:
I think is only called once when the command ends, so it should probably unconditionally call `turret.stop()`